src/quo2/components/settings/settings_list/style.cljs

```diff
@@ -21,6 +21,11 @@
    :padding-horizontal 12
    :padding-vertical   13})
 
+(def border-style
+  {:border-color (colors/theme-colors colors/neutral-20 colors/neutral-80)
+   :border-width 0.5
+   :overflow     :hidden
+  })
```

Comment:
Dangling parenthesis.

On a side note, I've noticed some devs face more of this problem because they don't do structural editing https://clojure.org/guides/structural_editing. Every major text editor has some form of support for this. Please ignore my comment if it doesn't apply to you.